tools/cldr-apps/WebContent/js/survey.js

```diff
@@ -2268,6 +2195,9 @@ function appendForumStuff(tr, theRow, forumDiv) {
 	var theForum = 	locmap.getLanguage(surveyCurrentLocale);
 	forumDiv.replyStub = contextPath + "/survey?forum=" + theForum + "&_=" + surveyCurrentLocale + "&replyto=";
 	forumDiv.postUrl = forumDiv.replyStub + "x"+theRow;
+	/*
+	 * TODO: there is normally a "what" parameter for SurveyAjax; why missing here?
```

Comment:
Good question. It's used at https://github.com/unicode-org/cldr/pull/47/files#diff-c5fe99a1fec3813edc83d825bc12d9c9R2169 in loading the forum_count